selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -670,7 +700,8 @@ public void waitNoGitChangeMarkers() {
   }
 
   /**
-   * Wait for Git insertion marker in the opened editor.
+   * Waits for Git insertion marker between {@code startLine} and {@code endLine} including this two
```

Comment:
it could be written simple: `between the specified fromLine, inclusive, and toLine, inclusive.`